error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -889,6 +889,35 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /** Prefer {@link Flux#mapNotNull(Function)} over more contrived alternatives. */
+  abstract static class FluxMapNotNullMapOptional<T, S> {
```

Comment:
We're still figuring out our preferred naming strategy, but tentatively it'd be something like `FluxMapNotNullOrElse`. Since the rule below would yield the same name, perhaps we'd go for `FluxMapNotNullTransformationOrElse` here :thinking:. Will use that for now.